clients/cloud/python/README.md

```diff
@@ -17,6 +17,24 @@ sasl.username=<api-key-id>
 sasl.password=<secret-access-key>
 ```
 
+* Properly configured trust store on client system
+
+In order for the producer to verify the certificate of the brokers make sure the trust store is set up correctly on your system. 
+
+** CentOS
```

Comment:
Use `### CentOS` for proper markdown rendering.

Reply:
Fixed